app/manifest.json

```diff
@@ -32,14 +32,6 @@
     ],
     "persistent": true
   },
-  "browser_action": {
```

Comment:
These changes we can do via webpack, I think I already have a commit that removes it actually.